airflow/jobs.py

```diff
@@ -28,6 +28,7 @@
 import sys
 import threading
 import time
+import json
```

Comment:
```suggestion
try:
    import ujson as json
except ImportError:
    import json
```

Reply:
I don't actually need this line anymore, so I just deleted it.